src/components/with-api-handler/index.js

```diff
@@ -32,7 +40,15 @@ export default () =>
 						} )
 						.catch( error => {
 							const { message } = error;
-							getNotices().forEach( notice => removeNotice( notice.id ) );
+							getNotices().forEach( notice => {
+								// Don't remove the "Campaign sent" notice.
+								if (
+									'success' !== notice.status ||
+									-1 === notice.content.indexOf( 'Campaign sent' )
```

Comment:
Kind of a nitpick, but this would break if a translation is applied – a variable should be used, storing the value of `__( 'Campaign sent on ', 'newspack-newsletters' )`

Reply:
> A general question: why are there changes to `composer.lock` file? I assume that file should not change unless `composer.json` changes.

I'm not sure why there were changes—perhaps because of a difference in the version of Composer I have installed? I didn't change anything in `composer.json`, but running `composer install` (since this was my first time firing up dev for this repo) resulted in these updates, so I committed them. The changes seem benign to me, but I do see a few packages have updated version numbers. Again, not sure why that happened.

> Kind of a nitpick, but this would break if a translation is applied – a variable should be used, storing the value of __( 'Campaign sent on ', 'newspack-newsletters' )

Good catch, I'll push a fix.

Reply:
@adekbadek Fixed this in https://github.com/Automattic/newspack-newsletters/pull/244/commits/5638aee41caaa4a368b3513c4dfbe4fd1fc759f9